### ISSUE TITLE
very Slow initial model.fit in Win10  

### ISSUE THREAD

**Issue by sniperlucian** [plaidml/plaidml#1117]
I am using the Rx 5700 on linux and win. The initial setup of the model in VRAM seems to takes a very long time on win compared to linux at the same model and data. 

What I can see is that the win task manager reports ~10MB/s disk activity (but resoucemonitor only shows 1MB/s), where I don't know what happens in the background. (Registry access?) Once this is past - the speed is comparable to linux.

Win10 is sitting in a QEMU GPU passthrough configuration and I dont have any performance issues when it comes to gaming compared to raw metal. Due to the NAVI reset bug I would like to be able to use plainml in Win10 -since it is a big pain to switch it between linux host and win guest.

Any advice how to debug this?   Are here any background disk activity from plainml I could speed up?

                                        linux                  win10

                 compile Time            1ms                     3ms
                 first Epoch             ~ sec                  188 sec      !!
                 n     Epoch             < 1 sec                < 1 sec 
                 all together           ~ 94 sec               ~280 sec  

GPU:  Rx 5700 
OS:  Win10 / Manjaro 

Model:
Shape of Data: [1,132,950]

model = Sequential()  
model.add(LSTM(950, input_shape=(nn,data.shape[2]), unroll=True, return_sequences=True))
model.add(Dense(450))
model.add(Dense(150))
model.add(Dense(1))
tt= time.time()
model.compile(loss='mean_absolute_error', optimizer='adam')
print("compile at",time.time()-tt)
model.fit(data, target, epochs=1000, batch_size=1, verbose=2)
print("finished after",time.time()-tt)

**Comment by Gerzer** [comment]
It's a known issue that LSTM (and GRU, *etc.*) training in PlaidML is *very* slow currently. Last I heard, the developers plan to address the issue once the transition to MLIR is complete—though that transition seems to be taking a long time.

**Comment by masc-it** [comment]
I don't think it's strictly related to LSTMs.  I have the same slow down on the 1st step and other random slow downs during training with every model I use (random high disk usage). On linux is just smooth.
